go/vt/vtgate/safe_session.go

```diff
@@ -166,6 +187,33 @@ func (session *SafeSession) AutocommitApproval() bool {
 	return false
 }
 
+// SetSavepointState sets the state to insertSavepoints if true and noInsertSavepoints if false
```

Comment:
```suggestion
// SetSavepointState sets the state to insertSavepoints if true and noInsertSavepoints is false
```

Reply:
the statement is correct, suggestion means that it is not clear, will rephrase it.